src/frontend-admin/src/components/HelloUser/HelloUser.module.css

```diff
@@ -0,0 +1,30 @@
+.helloMain {
+    position: absolute;
+    display: flex;
+    align-items: center;
+    right: 75px;
+    top: 0px;
+    height: 60px;
+}
+
+.helloMessage {
+    text-transform: capitalize;
+    line-height: 60px;
+
+    color: #333333;		
+    font-size: 16px;	
+    font-weight: 500;	
+    letter-spacing: -0.52px;
+    text-align: right; 
+}
+
+.dropdownArrow {	
```

Comment:
A pattern to start looking out for: any time you have styles in your css modules that are duplicated in other css modules, a new reusable component may be appropriate. I _think_ I've seen these styles (or something similar) in your dropdown selector. Maybe a `DropdownArrow` component with its own css module would be appropriate?

Reply:
I was going to remove it completely when the drop down is added. It was only there so I could see how it would look